india_stocks_api/internal/fivepaisa/api/order_api.py

```
@@ -0,0 +1,636 @@
+from india_stocks_api.internal.fivepaisa.mapping.transform_data import transform_data, map_product_type, reverse_map_product_type, transform_modify_order_data
+from india_stocks_api.internal.fivepaisa.mapping.transform_data import map_exchange, map_exchange_type, reverse_map_exchange
+
+logger = get_logger(__name__)
+
+
+# Base URL for 5Paisa API
+BASE_URL = "https://Openapi.5paisa.com"
+
+creds=get_credentials("fivepaisa")
+api_key = 'creds["api_key"]'
+api_secret = 'creds["api_secret"]'
```

Ruff (0.14.14)

Possible hardcoded password assigned to: "api_secret"

(S105)